### DISCUSSION TITLE
Bar gauge "max height"

### DISCUSSION THREAD

**Discussion by ant-mermaid-street** [grafana/grafana#49386] [Dashboards, Panels & Visualizations]
Hello
Currently we only have the option to set the "Min height" on the Bar gauge panel, please add a new option for the "Max height".
use case:
Imagine a panel that can display one or 40 items depending on the variable you select, when leaving the "Min height" on auto it will show pretty nice, something like that:
![image](https://user-images.githubusercontent.com/22320901/169779061-c4be92ae-4097-4ce4-8671-709a3fd95262.png)

but if you select just several items, it will be very ugly:
![image](https://user-images.githubusercontent.com/22320901/169779589-46b44e59-53c2-40b6-b510-5d78b5bfff70.png)

please add the option for the "Max height"
Thank you.

**Comment by nmarrs** [comment]
Hi! We have created [this github issue](https://github.com/grafana/grafana/issues/75982) to track this request. Thanks!
